packages/www/pages/app/stream/[id].tsx

```diff
@@ -257,6 +259,10 @@ const ID = () => {
         throw new Error(info.errors.toString());
       }
       setStream(info.stream);
+      if (stream.playbackPolicy?.type != "public" && info.stream.playbackId) {
```

Comment:
Can we tell if the user is an admin here? Would be nice to avoid this logic if it's not an admin

Reply:
Isn't this the admin UI anyway? Non admin users can never access this

Reply:
I don't think so. It is just the old dashboard, and it shows additional information if the user is admin (so maybe we can tell that here somehow 🤔 )

Reply:
Old dashboard? :open_mouth: This is new to me lol - added the admin check!